MEIOUandTaxes1/src/common/scripted_effects/SYS-StartUp_Produce.txt

```diff
@@ -36,7 +36,46 @@ StartUp_InitResource = {
 				}
 				set_province_flag = cotton
 			}
-
+			if = { #making sure austronesia has spices
+				limit = {
+					AND = {
```

Comment:
This code is imo very unnecessarily complicated. Standard logical scope of any trigger is AND so the first AND can be removed. Besides that perhaps this big NOT => OR logic should be moved into a scriped trigger.